src/Pretzel/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Pretzel": {
       "commandName": "Project",
-      "commandLineArgs": "taste -s c:\\f\\git\\Scissors.Blog --debug"
+      "commandLineArgs": "taste -s c:\\f\\git\\Scissors.Blog"
```

Comment:
Sorry I haven't seen this earlier, but shouldn't the command line used a relative path in order to be used by anyone?

Reply:
I did notice either. Either we should delete and ignore the File all together, or provide a sample Project by Default